smokey/src/main/java/frc/robot/commands/AutoBalanceStepCommand.java

```diff
@@ -119,13 +124,19 @@ public boolean isFinished()
 
   /**
    * Scales XY values to the desired velocity 
-   * @param vec - Translation2d vector to be scaled
+   * @param angles - Translation2d vector representing angle of steepest ascent
    * @return - Translation2d scaled vector
    */
-  private Translation2d normalizeXYVelocities(Translation2d vec)
+  private Translation2d normalizeXYVelocities(Translation2d angles)
   {
-    double h = Math.sqrt(Math.pow(vec.getX(), 2) + Math.pow(vec.getY(), 2));
-    return new Translation2d((vec.getX()/h) * velocityValue, (vec.getY()/h) * velocityValue);
+    double h = Math.sqrt(Math.pow(angles.getX(), 2) + Math.pow(angles.getY(), 2));
+    double velocity = 0;
+    // we want positive velcoty when the error is positive
+    velocity = -pidController.calculate(h, 0.0);
+    velocity = MotorUtils.clamp(velocity, 0.0, velocityValue);
+    System.out.println("Setting velcoty to " + velocity + " for angle error " + h);
```

Comment:
spelling and I presume we might delete this later as this print might spew ... not 100% how often this method is called

Reply:
fixed spelling.  This is only called at the end of each iteration, so ~3-12 times per balance attempt.  It will be useful if the balancing doesn't work in a match, we can look back and diagnose what happened.